client/my-sites/plans/jetpack-plans/use-item-price.ts

```diff
@@ -149,7 +149,7 @@ const useItemPrice = (
 		if ( item.term !== TERM_MONTHLY ) {
 			originalPrice = monthlyItemCost ?? itemCost / 12;
 			discountedPrice = introductoryOfferPrices.introOfferCost
-				? introductoryOfferPrices.introOfferCost / 12
+				? ( introductoryOfferPrices.introOfferCost * 100 ) / 12 / 100
```

Comment:
This might re-introduce some early rounding issues we ran into from https://github.com/Automattic/wp-calypso/pull/61379, where we were rounding up the intro offer cost prior to an additional discount potentially being applied, leading to inconsistent rounding of the discount percentage badges.

Although, if we're moving towards specific discount percentages for each product, this has less of an impact. If we did keep rounding here, I think we should also round the `originalPrice` as well to keep the results of `useItemPrice()` consistent. What do you think?

Reply:
yup, we should definitely round `originalPrice` in the same way. I added this because even though the result of `introductoryOfferPrices.introOfferCost / 12` was chosen to be evenly divisible, it was not producing a even result. Let me see if we have other options for a fix...

Reply:
Looking at #61379 and the original `useMonthlyPrice` function, the key difference is not applying a rounding function. If we only multiply and divide by `100`, the fractional element of an uneven division is preserved to be used for rounding later.
<img width="181" alt="Screen Shot 2022-03-10 at 10 30 01" src="https://user-images.githubusercontent.com/2810519/157731047-49ae30ba-07e3-4528-a06d-64a7fb04e01d.png">

These console logs compare the `* 100 / 12 / 100` logic versus `12`. In the case of $59.95, the monthly price still has the fraction, where $47.40 is cleanly divided.

Reply:
Oh, perfect. I understand what's going on now. My brain seems to have just auto-associated the * 100 / 100 pattern as a rounding scenario from working on this previously 😛